DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
```

Comment:
I agree it is a bit confusing. This is what they are:

`_ExternalLoginsPartial`: the login methods that are shown on the manage logins page
`_ExternalLoginMethodsPartial`: the login methods that are shown on the login menu
`_ExternalLoginButtonsPartial`: the login methods that are shown on the login page

They are all slightly different in what they do and how they display.